index.html

```diff
@@ -97,7 +104,7 @@ <h1 class="work-for-you">РАБОТЫ, КОТОРЫЕ МЫ МОЖЕМ ВЫПОЛ
         </div>
     </div>
     <div class="footer">
-        <img src="./img/plumber-footer.webp" alt="logo">
+        <img class="footer-logo" src="./img/plumber-footer.webp" alt="logo" style: width="100px">
```

Comment:
Please move style to css. Moreover, this usage is invalid. Always check the validity of the code (and the result in the browser)!